### DISCUSSION TITLE
Compile Error

### DISCUSSION THREAD

**Discussion by pa-one55** [AMReX-Codes/amrex#4137] [Q&A]
![image](https://github.com/user-attachments/assets/7af7a0ed-57cd-41f8-98bb-60128ebc2a7c)

what is this error ?
i went inside - amrex/amrex-tutorials/GuidedTutorials/HeatEquation_Simple
and did : 
mkdir build
cd build
cmake ..
make
and got this error

what do i do now
something similar is happening on gitpod as well.

when i directly did - make
i got the error saying coulnt find - #include <AMReX.H>

**Comment by WeiqunZhang** [comment]
Where did you the source code? Could you tell us step by step what you did starting from `git clone`?

I cannot find any occurrences of  `computeNewState` in amrex or amrex-tutorial.
